src/renderer/src/libs/walletConnectSDK.ts

```diff
@@ -0,0 +1,73 @@
+import { AbstractWalletConnectNeonAdapter } from '@cityofzion/wallet-connect-sdk-wallet-core'
+import { TAdapterMethodParam } from '@cityofzion/wallet-connect-sdk-wallet-react'
+import { TOptions } from '@cityofzion/wallet-connect-sdk-wallet-react'
+import { WalletConnectHelper } from '@renderer/helpers/WalletConnectHelper'
+import { RootStore } from '@renderer/store/RootStore'
+import i18n from 'i18next'
+export class WalletConnectNeonAdapter extends AbstractWalletConnectNeonAdapter {
+  async getAccountString({ session }: TAdapterMethodParam): Promise<string> {
+    const {
+      account: { data: accounts },
+      settings: { encryptedPassword },
+    } = RootStore.store.getState()
+
+    const [{ address }] = WalletConnectHelper.getAccountInformationFromSession(session)
+    const account = accounts.find(account => account.address === address)
+    if (!account) throw new Error('Account not found')
+    if (!account.encryptedKey) throw new Error('Key not found')
+
+    const key = await window.api.decryptBasedEncryptedSecret(account.encryptedKey, encryptedPassword)
+    if (!key) throw new Error('Error to decrypt key')
+
+    return key
+  }
+
+  async getWalletInfo(): Promise<any> {
+    // TODO: Implement this method when ledger is supported
```

Comment:
every TODO comment needs an issue code, otherwise it will be forgotten

Reply:
If you think this is not a real problem, ask Mirella and Lucas, but warning: they might scream at you LOL JK